source/client-side-encryption/tests/README.rst

```diff
@@ -349,7 +349,7 @@ for information on "disabling" csfle and setting csfle search paths.
 
    .. code-block:: shell
 
-      $ python3 mongodl.py --component=csfle --version=5.3.1 --out=./csfle/
+      $ python3 mongodl.py --component=csfle --version=6.0.0-rc4 --out=./csfle/
```

Comment:
Can we have a note here about the availability of newer/older versions? I don't want to imply that this example shows "the correct version" to be used.